src/index.html

```diff
@@ -9,6 +9,23 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <div class="logo">
+          <div class="logo__img"> <a href="#home"><img src="images/logo.png" alt="moyo-logo"></a></div>
```

Comment:
Please check your formatting, tags should be placed as in the picture, with two spaces indented from the parent element to the child
![image](https://github.com/mate-academy/layout_moyo-header/assets/54936044/7b64e300-a1a4-4235-8637-b89914323998)